client/src/carousel.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// === STYLES === //
+
+const Wrapper = styled.div`
+  width: 1280px;
+  overflow: hidden;
+`;
+
+const CarouselContainer = styled.div`
+  display: flex;
+  margin: 0 0 20px 20 px;
+  align-content: center;
+`;
+
+// flex: 0 (flex-grow off) 1 (flex shrink on )
+const CarouselSlot = styled.div`
+  flex: 0 1 100%;
+  flex-basis: 80%;
+  margin-right: 20px;
+  order: ${(props) => props.order};
+`;
+
+const Title = styled.div`
+  font-family: Alternate Gothic No1 D;
+  font-size: 20px;
+  color: #4c4c4c;
+`;
+
+const Role = styled.div`
+  font-family: "Times New Roman";
+  font-style: italic;
+  color: #4c4c4c;
+`;
+
+// ===  COMPONENT DEFINITION ===  //
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      position: 0
+    };
+  }
+
+  getOrder(itemIndex) {
+    const { position } = this.state;
```

Comment:
Object destructuring is great.

Reply:
just now trying it out.  I was sticking with using the props keyword till I got it down.